cfpb/consumerfinance.gov
# GitHub Actions filter update

Recent changes to our docker files made changes that broke tests that hadn't run on the PR as the file filters didn't cover any changes to the Dockefile.
This PR adds Dockerfile and related conf files to the list of triggeers for both backend and front end tests. I also added CSV to the Backend tests as some back end test do rely on csv files so its important to make sure any changes are still working with the tests.


--

## Additions

-new file types added to the filters of both front and back end GitHub Actions



## How to test this PR

1.This is a bit more difficult to test, as it only really applies to future PR's that affect a certain file type. One could add a few lines to said files in thisPR see if the PR runs the tests and then revert the changes


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
